dqc_us_rules/dqc_us_0006.py

```diff
@@ -75,6 +75,20 @@ def _dict_list_update(dict_a, dict_b):
         dict_a[key].extend(val)
     return dict_a
 
```

Comment:
however, should add an extra line here to make two blank lines above this function